AppController/lib/zkinterface.rb

```diff
@@ -130,12 +116,7 @@ def self.remove_revision_entry(revision_key, ip)
   end
 
   def self.get_revision_hosters(revision_key, keyname)
-    revision_hosters = get_children("#{ROOT_APP_PATH}/#{revision_key}")
-    converted = []
-    revision_hosters.each { |host|
-      converted << NodeInfo.new(get_job_data_for_ip(host), keyname)
-    }
-    converted
+    get_children("#{ROOT_APP_PATH}/#{revision_key}")
```

Comment:
`keyname` argument doesn't seem to be used any more.
Unless it's ruby best practice, could you also add explicit `return` statement?

Reply:
It's somewhat best practices ... but we also do both in that file ... I will add it since you asked nicely ROFL